README.md

```diff
@@ -188,6 +191,19 @@ Accepted values:
 The generated account keys are preserved across reloads, but will be lost on
 restart unless [state_path](#state_path) is configured.
 
+### challenge
```

Comment:
Why it cannot be enum?
What if I want to say I want to accept any challenge the module is able to solve?